<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifierMdpTech</class>
 <widget class="QDialog" name="modifierMdpTech">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>159</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>384</width>
    <height>159</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>384</width>
    <height>159</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestion Technicien - Modifier Mot De Passe</string>
  </property>
  <widget class="QLineEdit" name="mdpAdmin">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Mot de passe admin</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginTech">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Login technicien</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nouveauMdp">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Nouveau mot de passe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generer">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Générer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifier">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="annuler">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
